backend/core/utils/__init__.py

```diff
@@ -226,6 +226,11 @@ def purge_redis_eph_pers_keys(redis_instance, ephemeral_key, persistent_key):
     redis_pipeline.delete(ephemeral_key)
     redis_pipeline.srem("persistent-keys", persistent_key)
     redis_pipeline.delete("hij_orig_neighb_{}".format(ephemeral_key))
+    for element in redis_instance.sscan_iter(
+        "hij_prefix_peer_{}".format(ephemeral_key)
+    ):
+        redis_pipeline.srem("hij_prefix_peer_all", element)
```

Comment:
TODO: make sure that this combination is not present in any other ongoing hijack for the same prefix (of different type) with this peer. However, this is extremely rare. I have a way to deal with this by inputting (in redis) sets of hijack keys with the name: hij_prefix_<prefix>_peer_<peer>. Will fix soon, should not be very hard.